### DISCUSSION TITLE
posts/silversharkmod/

### DISCUSSION THREAD

**Discussion by giscus** [huangluzhi/huangluzhi.github.io#2] [Announcements]
# posts/silversharkmod/

银鲨Silvershark VII 的 Assetto Corsa mod 制作过程的回顾纪录。可供其他mod制作者提供参考借鉴，避免一些初级错误。

http://localhost:4001/posts/silversharkmod/

**Comment by huangluzhi** [comment]
Update: I'm no longer working with AC mod and accepting any mod content making request unless I make a new annoucement. But you are still welcome to contact me if you have any questions :)

**Comment by ACR20927** [comment]
感谢北理学长
